packages/astro/src/content/template/virtual-mod.mjs

```diff
@@ -11,6 +11,13 @@ export function defineCollection(config) {
 	return config;
 }
 
+// TODO: Remove this when having this fallback is no longer relevant. 2.3? 3.0? - erika, 2023-04-04
+export const image = () => {
+	throw new Error(
+		'Importing image() from `astro:content` is no longer supported. See https://docs.astro.build/en/guides/assets/#update-content-collections-schemas for more information.'
```

Comment:
```suggestion
		'Importing `image()` from `astro:content` is no longer supported. See https://docs.astro.build/en/guides/assets/#update-content-collections-schemas for our new import instructions.'
```

Nit: new wording to make it clear there's a _new_ way to do it, and the feature didn't disappear